toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -729,6 +730,29 @@ func InstallGrubCfg(installRoot, rootDevice, bootUUID string, encryptedRoot disk
 		return
 	}
 
+	// Configure IMA policy
+	err = setGrubCfgIMA(installGrubCfgFile, kernelCommandLine)
+	if err != nil {
+		logger.Log.Warnf("Failed to set ima_policy in grub.cfg: %v", err)
+		return
+	}
+
+	// Append any additional command line parameters
+	err = setGrubCfgAdditional(installGrubCfgFile, kernelCommandLine)
+	if err != nil {
+		logger.Log.Warnf("Failed to append extra command line parameterse in grub.cfg: %v", err)
+		return
+	}
+
+	logger.Log.Errorf("Grub at %s", installGrubCfgFile)
```

Comment:
Debugf or Tracef. Applies to the other logger messages below too.

Reply:
Just going to strip them out completely, they are no longer needed.